### PR TITLE
Added another Hackathon.

### PR DESCRIPTION
**Pull request by udit-001**
#7

### CODE REVIEW

**Comment by geekyd** [comment]
@udit-001 Also resolve the conflicts

**Comment by udit-001** [comment]
Hey! I resolved conflicts as well as fixed the format, the changes seem ready to be merged with the master !

File: README.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
-# Event Buddy
+# ![alt Event Buddy](/src/assets/logo.png)
```
Comment (geekyd) [inline README.md]: @udit-001 These commits have already been merged to the master. Please clear your history. While pulling from upstream use `git pull --rebase upstream master`

Comment (udit-001) [reply README.md]: I don't understand how to proceed further, perhaps a more detailed instruction would help. ;/

Comment (geekyd) [reply README.md]: Run `git rebase -i HEAD~2` command and delete the commit that is already merged & then pull from upstream using `git pull --rebase upstream`

Comment (udit-001) [reply README.md]: @geekyd It seems good to go now!

**Pull request merged**